### PR TITLE
Default form updates

### PR DESCRIPTION
**Pull request by lfarrell**
https://jira.lib.unc.edu/browse/HYC-436

### CODE REVIEW

File: config/locales/hyrax.en.yml [hunk 1]
```diff
@@ -192,15 +192,15 @@ en:
...
         advisor: "Your advisor's name. Please enter the last name first, e.g. \"Smith, John.\""
-        affiliation: "Your primary departmental affiliation at UNC"
+        affiliation: "Primary departmental affiliation"
         alternative_title: "A unique secondary title for your work if applicable. Enter subtitle in the \"Title\" field, instead."
         award: "Honors level"
         bibliographic_citation: "A citation to a related work, or a different version of this work."
         contributor: "A person or group who had a role in creating the work."
         copyright_date: "The date that the work was copyrighted. This may be different than today's date. If you aren't sure, leave this blank."
         creator: "The person or group responsible for creating the work. If you are depositing a paper, article or book chapter, enter the author's name"
         date_created: "The date on which the work was created. If you are depositing a thesis, enter the date that it was completed."
```
Comment (smithjp) [inline config/locales/hyrax.en.yml]: Thesis text still shows up for all forms.  Is it possible to remove from some forms?

File: app/forms/hyrax/general_form.rb [hunk 2]
```diff
@@ -8,20 +8,23 @@ class GeneralForm < ::SingleValueForm
     self.model_class = ::General
 
     self.terms += [:resource_type, :bibliographic_citation, :abstract, :academic_concentration, :access, :advisor,
```
Comment (smithjp) [inline app/forms/hyrax/general_form.rb]: advisor fields are not grouped with rest of people fields

File: config/locales/hyrax.en.yml (hunk 1 above)
Comment (lfarrell) [reply config/locales/hyrax.en.yml]: Looks like it was in multiple fields. All the help text should be up to date now.

File: app/forms/hyrax/general_form.rb (hunk 2 above)
Comment (lfarrell) [reply app/forms/hyrax/general_form.rb]: Ugh, updated. :)

**Pull request merged**